float32 k_p
float32 k_i
float32 k_d
float32 basic_gain
uint8 sys_id
---
bool success